src/main/webapp/index.html

```diff
@@ -22,133 +22,125 @@
     
     <!--Title -->
     <div id="moduleTitle">
-        <img src="images/logo.png">
-        <div id="titleText">
-          <h1 class="bigger">Dataflow</h1>
-          <h1> Service Analyser</h1>
-        </div>
+      <img src="images/logo.png">
+      <div id="titleText">
+        <h1 class="bigger">Dataflow</h1>
+        <h1> Service Analyser</h1>
+      </div>
     </div>
 
     <div class="moduleSpace"></div>
-    <a id="returnTop">
-    <!-- Input for the Project ID-->
-    <form id="projectIDForm">
-      <input type="text" id="projectID" placeholder="Enter Project ID" name="projectID" required>
-      <input type="button" onclick="updateProjectDatabase()" value = "Fetch">
-      <!-- Button visible when there are some permissions missing -->
-      <button 
-          class="permissionButton" 
-          id="showMissingPermision" 
-          onmouseover="document.getElementById('permissionDialog').hidden = false;" 
-          onmouseout="document.getElementById('permissionDialog').hidden = true;"
-          hidden>
-      </button>
-      <!-- Button visible when all permission are set up -->
-      <button 
-          class="permissionButton" 
-          id="allPermisionsCorrect" 
-          onmouseover="document.getElementById('permissionDialog').hidden = false;"
-          onmouseout="document.getElementById('permissionDialog').hidden = true;"
-          hidden>
-      </button> 
-    </form>
+    
+    <!-- The user must provide the right information in order to be able to have access to the project -->
+    <div id="access">
+      <form id="accessForm" name="acessForm">
+        <input type="text" id="projectIdAccess" placeholder="Enter Project ID" name="projectIdAccess" required>
+        <input type="text" id="bucketName" placeholder="Enter Bucket Name" name="bucketName" required>
+        <input type="text" id="objectName" placeholder="Enter Object Name" name="objectName" required>
+      </form>
+      <div class="accessContainer">
+        <h2> Let's start </h2>
+        <button id="accessButton" onclick="accessFunction()"> </button>
+      </div>
+    </div>
 
-    <!-- Pops up when the user hover over a button from the permissionButton class -->
-    <div id="permissionDialog" hidden></div>
+    <!-- The content is only visible after the access information is provided -->
+    <div id="content">
+      <div class="moduleSpace"></div>
+      <a id="returnTop">
+      <!-- Input for the Project ID-->
+      <form id="projectIDForm">
+        <input type="text" id="projectID" placeholder="Enter Project ID" name="projectID" disabled="disabled" required>
```

Comment:
Is this still needed? Just because they've already provided their projectID so we could use that, and if they wanted to switch project we would need them to back through the whole same process (i.e. we couldn't just change the project ID).

Reply:
The typing from the user is disabled as I use it just to show the current project we are working on. I think we should leave it as it is, as it provides an overall better looking user interface

Reply:
Ah okay I hadn't noticed that it was disabled :)

Reply:
No problem :D Glad we discussed it.